### ISSUE TITLE
Add multilanguage GUI

### ISSUE THREAD

**Issue by bovirus** [jlw4049/MP4-Mux-Tool#25]
@jlw4049 

Could you please modify source code to have an external file with list of strings to enable a multilanguage GUI?

Thanks.

**Comment by jlw4049** [comment]
I'll keep this issue up. During the re write I'll look into this!

**Comment by bovirus** [comment]
@jlw4049 

If you will switch to multilanguage I can help with Italian language.
If you can use separate language file for each language.
Thanks.

**Comment by jlw4049** [comment]
> @jlw4049 
> 
> If you will switch to multilanguage I can help with Italian language.
> If you can use separate language file for each language.
> Thanks.
I'm re writing the tool and I'll keep this on mind if you're still interested.

**Comment by bovirus** [comment]
@jlw4049 

Does it mean that you could consider my request?

**Comment by jlw4049** [comment]
> @jlw4049
> 
> Does it mean that you could consider my request?

Definitely. I'm re-writing the application from the ground up. I had to break off to work on a couple other projects, so progress has been a little on the slow slide right now. The work is being done in this branch [here](https://github.com/jlw4049/MP4-Mux-Tool/tree/qt-re-work). You can follow progress if you want, this will replace the old one. 

It'll have new features and adding multi-language would be nice.

**Comment by bovirus** [comment]
@jlw4049 

When you will have the file to tranlsate post the info and file path and I will translate them.

Thanks.

**Comment by jlw4049** [comment]
> @jlw4049
> 
> When you will have the file to tranlsate post the info and file path and I will translate them.
> 
> Thanks.

Awesome I'll keep this issue opened and when we get there I'll reach out to you here!
